deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node-secure-for-test.yaml

```diff
@@ -0,0 +1,307 @@
+# Setup Service to provide access to Zookeeper for clients
```

Comment:
Please consider using `zookeeper-1-node-1GB-for-tests-only-scaleout-pvc.yaml` as a template